ex1/ex1.c

```diff
@@ -9,6 +9,15 @@
 int main(int argc, char *argv[])
 {
     // Your code here
+    char str[100] = "What does fork() do\n";
+    int num = 100;
+    printf(str);
+    printf(num)
+    fork();
+    str = "Because it has been forked, this sentence should print twice.\n";
```

Comment:
I had to #include <string.h> at the top and change this line to

strcpy(str, "Because it has been forked, this sentence should print twice.\n");

since str can't be changed by saying str = "Because it has been forked, this sentence should print twice.\n"